src/Core_Screencap/Core.ScreenshotManager.cs

```diff
@@ -72,6 +73,14 @@ public partial class ScreenshotManager
         public static ConfigEntry<CameraGuideLinesMode> GuideLinesModes { get; private set; }
         public static ConfigEntry<int> UIShotUpscale { get; private set; }
 
+        // screenshot setting presets
+
+        public static ConfigEntry<int> presetCount { get; private set; }
+        public static List<ConfigEntry<KeyboardShortcut>> presetKeys { get; private set; } = new List<ConfigEntry<KeyboardShortcut>>();
+        public static List<ConfigEntry<int>> presetResX { get; private set; } = new List<ConfigEntry<int>>();
+        public static List<ConfigEntry<int>> presetResY { get; private set; } = new List<ConfigEntry<int>>();
+        public static List<ConfigEntry<int>> presetDownscaling { get; private set; } = new List<ConfigEntry<int>>();
```

Comment:
I'd rather make a struct to hold all these config entries in a single list.

Reply:
Understandable, I can fix that 😄